### PR TITLE
Update changelog for 0.13.0 release

### PR DESCRIPTION
**Pull request by yuryybk**

### CODE REVIEW

**Comment by yuryybk** [comment]
@yunikkk @kiryldz please compare with https://github.com/mapbox/mapbox-vision/blob/release/0.13.0/docs/changelog.md if I've described changelog correctly

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest.
+- Added selection of default 960x540 camera resolution or closest to 16x9 in Camera2VideoSourceImpl
+- Added param to Camera2VideoSourceImpl constructor to enable\disable autofocus
+- Added support of reverse landscape orientation
+- Updated SNPE to 1.39.0
+- Removed deprecated code for 0.12.0
+
+### AR
+- Replaced `VisionView` rendering with common rendering engine for all platforms.
```
Comment (kiryldz) [inline CHANGELOG.md]: This is `Vision`, not `AR`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+## v0.13.0
+
+### Vision
+- Added video recording for custom `VideoSource` implementation
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest.
+- Added selection of default 960x540 camera resolution or closest to 16x9 in Camera2VideoSourceImpl
+- Added param to Camera2VideoSourceImpl constructor to enable\disable autofocus
+- Added support of reverse landscape orientation
+- Updated SNPE to 1.39.0
```
Comment (kiryldz) [inline CHANGELOG.md]: Kind of tricky moment as we provide public build with Mace.
@yunikkk WDYT?

**Comment by kiryldz** [comment]
@yuryybk, where is `Integrated LaneDetection mode for VisionView` BTW?
As well as 
`Deprecate VisionArEventsListener.onArCameraUpdated(ArCamera) method`
`Extend Camera class returned by VisionEventsListener.onCameraUpdated`

File: CHANGELOG.md (hunk 2 above)
Comment (yunikkk) [reply CHANGELOG.md]: Agree, this is not needed in public changelog

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## v0.13.0
+
+### Vision
+- Added video recording for custom `VideoSource` implementation
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest.
+- Added selection of default 960x540 camera resolution or closest to 16x9 in Camera2VideoSourceImpl
```
Comment (yunikkk) [inline CHANGELOG.md]: Nit : `Changed default resolution of Camera2VideoSourceImpl to 960*540`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## v0.13.0
+
+### Vision
+- Added video recording for custom `VideoSource` implementation
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest.
```
Comment (yunikkk) [inline CHANGELOG.md]: nit : dot not needed in the end

File: CHANGELOG.md (hunk 1 above)
Comment (yunikkk) [reply CHANGELOG.md]: nit : remove dots

Comment (yuryybk) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (yuryybk) [reply CHANGELOG.md]: okay, will remove

File: CHANGELOG.md (hunk 3 above)
Comment (yuryybk) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 4 above)
Comment (yuryybk) [reply CHANGELOG.md]: fixed

**Comment by yuryybk** [comment]
> @yuryybk, where is `Integrated LaneDetection mode for VisionView` BTW?
> As well as
> `Deprecate VisionArEventsListener.onArCameraUpdated(ArCamera) method`
> `Extend Camera class returned by VisionEventsListener.onCameraUpdated`

Added to changelog

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,21 @@
...
+## v0.13.0
+
+### Vision
+- Added video recording for custom `VideoSource` implementation
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest
+- Changed default resolution of Camera2VideoSourceImpl to 960*540
+- Added param to Camera2VideoSourceImpl constructor to enable\disable autofocus
+- Added support of reverse landscape orientation
+- Removed deprecated code for 0.12.0
```
Comment (yunikkk) [inline CHANGELOG.md]: I'd listed specific deprecated code that was removed

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,21 @@
...
+- Replaced `VisionView` rendering with common rendering engine for all platforms
+- Replaced `VisionArView` rendering with common rendering engine for all platforms
+- Integrated LaneDetection mode for VisionView
```
Comment (yunikkk) [inline CHANGELOG.md]: BTW has iOS implemented the same? @kiryldz @chezzdev

Comment (chezzdev) [reply CHANGELOG.md]: No, since there's no VRE integrated, we don't have this mode.

**Comment by chezzdev** [comment]
Have we formed the general changelog first?

**Comment by yuryybk** [comment]
@chezzdev not sure exactly about the process, I've built it from the mapbox-vision/docs/changelog.md.
I've found an issue from the previous changelog, https://github.com/mapbox/mapbox-vision/issues/2314, we need to discuss the correct flow.

File: CHANGELOG.md (hunk 5 above)
Comment (yuryybk) [reply CHANGELOG.md]: Added, @kiryldz please check removed deprecated code as it is related to this PR 
https://github.com/mapbox/mapbox-vision/pull/2584

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## v0.13.0
+
+### Vision
+- Added video recording for custom `VideoSource` implementation
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest
+- Changed default resolution of Camera2VideoSourceImpl to 960*540
+- Added param to Camera2VideoSourceImpl constructor to enable\disable autofocus
```
Comment (kiryldz) [inline CHANGELOG.md]:
```suggestion
- Added optional parameter to Camera2VideoSourceImpl constructor to enable\disable autofocus
```

Comment (yuryybk) [reply CHANGELOG.md]: fixed

**Comment by yuryybk** [comment]
@chezzdev @yunikkk @kiryldz 
I've got changelog with the help of Dersim and tried to sort with Private\Public on Android
Please check changelog here and [autogenerated](https://github.com/mapbox/mapbox-vision/issues/2767#issuecomment-648785788)

**Review by kiryldz — APPROVED** [review]
LGTM from my side.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,30 @@
...
+  - `VisionReplayManager.setModelPerformanceConfig`
+  - `SystemInfoUtils.getSnpeSupportedBoard`
+  - `SystemInfoUtils.getSystemProperty`
+  - `enum class SupportedSnapdragonBoards`
+  - `class ModelPerformanceConfig`
+- Integrated new rendering engine for VisionView and VisionArView with better performance
+- Integrated LaneDetection mode for VisionView
+- Deprecated `VisionArEventsListener.onArCameraUpdated` method
+- Extended `Camera` class returned by `VisionEventsListener.onCameraUpdated`
+- Added Okinawa to Japan sign classifier
```
Comment (yunikkk) [inline CHANGELOG.md]: Guess not the sign classifier, but country detector itself.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,30 @@
...
+- Improved segmentation rendering
```
Comment (yunikkk) [inline CHANGELOG.md]: What are the improvements?

Comment (yuryybk) [reply CHANGELOG.md]: https://github.com/mapbox/mapbox-vision/pull/2554
Probably it doesn't deserve description

File: CHANGELOG.md (hunk 8 above)
Comment (yuryybk) [reply CHANGELOG.md]: changed

File: CHANGELOG.md (hunk 9 above)
Comment (yunikkk) [reply CHANGELOG.md]: I wouldn't list them then

**Review by yunikkk — APPROVED** [review]

File: CHANGELOG.md (hunk 9 above)
Comment (yuryybk) [reply CHANGELOG.md]: removed

**Comment by yuryybk** [comment]
Do not merge until we publish a new version of SDK

**Review by yunikkk — APPROVED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,32 @@
...
+- Add support of reverse landscape orientation
+- Integrated new rendering engine for VisionView and VisionArView with better performance
+- Add `VisionManager.setCameraHeight`
+- Deprecate `ARCamera` class
+- Add `aspectRatio`, `roll`, `pitch`, `yaw`, `height` properties to `Camera`
+- Improve lane detection
+- Stop sending some inaccurate events until the camera is calibrated
+- Introduce automatic camera recalibration
+- Expand Japan region to include Okinawa
+- Fix bug with speed estimation when a vehicle is stopped
```
Comment (kiryldz) [inline CHANGELOG.md:20]:
```suggestion
- Fix bug with speed estimation when a vehicle has stopped
```

**Review by kiryldz — APPROVED** [review]
Small nit

File: CHANGELOG.md (hunk 10 above)
Comment (yuryybk) [reply CHANGELOG.md:20]: applied

Comment (chezzdev) [reply CHANGELOG.md:20]: It's more about the state "stopped" than action "has stopped".

Comment (yuryybk) [reply CHANGELOG.md:20]: From my English point of view "has stopped" looks more correct too :)

Comment (kiryldz) [reply CHANGELOG.md:20]: @chezzdev perhaps `is not moving` then?

Comment (yuryybk) [reply CHANGELOG.md:20]: discussed, rolled back

**Review by chezzdev — APPROVED** [review]

**Pull request merged**